src/index.test.ts

```diff
@@ -199,9 +199,7 @@ describe('Instance', () => {
 
 describe('Response', () => {
   test('request should return `Response` object by default', async () => {
-    const scope = nock('http://localhost')
-      .get('/comments')
-      .reply(200)
+    const scope = nock('http://localhost').get('/comments').reply(200)
```

Comment:
I don't understand why formatting is changed here. Are your editor correctly picks up the project's `.prettierrc.js`?

Reply:
Hmm.. look like some issues indeed on my side. Ignore the commend :D